src/components/com_tjcertificate/administrator/controllers/certificates.php

```diff
@@ -34,4 +38,100 @@ public function getModel($name = 'Certificate', $prefix = 'TjCertificateModel')
 	{
 		return parent::getModel($name, $prefix, array('ignore_request' => true));
 	}
+
+	/**
+	 * Method to delete the certificate.
+	 *
+	 * @return  void
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function deleteCertificate()
+	{
+		$user = Factory::getUser();
+		$app  = Factory::getApplication();
+		$certificateId = $app->input->getInt('id');
+		$model = $this->getModel();
+
+		if (!$user->authorise('certificate.external.manage', 'com_tjcertificate'))
+		{
+			$checkCertificateExist = $model->checkCertificateExist($certificateId, $user->id);
```

Comment:
This code will only delete the logged-in users' certificate. but the permission you have checked for ADMIN i.e. 'certificate.external.manage'

Reply:
checked if not having admin permission then check if condition